src/core/const.js

```diff
@@ -1,400 +1,420 @@
 
-    /**
-     * Regexp for SVG size.
-     *
-     * @static
-     * @constant
-     * @type {RegExp|string}
-     * @example `<svg width="100" height="100"></svg>`
-     */
-    SVG_SIZE: /<svg[^>]*(?:\s(width|height)="(\d*(?:\.\d+)?)(?:px)?")[^>]*(?:\s(width|height)="(\d*(?:\.\d+)?)(?:px)?")[^>]*>/i,
+/**
+ * Regexp for data URI.
+ * Based on: https://github.com/ragingwind/data-uri-regex
+ *
+ * @static
+ * @constant
+ * @memberof PIXI
+ * @type {RegExp|string}
+ * @example `data:image/png;base64`
+ */
+export const DATA_URI = /^\s*data:(?:([\w-]+)\/([\w+.-]+))?(?:;(charset=[\w-]+|base64))?,(.*)/i;
 
-    /**
-     * Constants that identify shapes, mainly to prevent `instanceof` calls.
-     *
-     * @static
-     * @constant
-     * @type {object}
-     * @property {number} POLY
-     * @property {number} RECT
-     * @property {number} CIRC
-     * @property {number} ELIP
-     * @property {number} RREC
-     */
-    SHAPES: {
-        POLY: 0,
-        RECT: 1,
-        CIRC: 2,
-        ELIP: 3,
-        RREC: 4
-    },
+/**
+ * Regexp for SVG size.
+ *
+ * @static
+ * @constant
+ * @memberof PIXI
+ * @type {RegExp|string}
+ * @example `<svg width="100" height="100"></svg>`
+ */
+export const SVG_SIZE = /<svg[^>]*(?:\s(width|height)="(\d*(?:\.\d+)?)(?:px)?")[^>]*(?:\s(width|height)="(\d*(?:\.\d+)?)(?:px)?")[^>]*>/i; // eslint-disable-line max-len
 
-    /**
-     * Constants that specify float precision in shaders.
-     *
-     * @static
-     * @constant
-     * @type {object}
-     * @property {number} DEFAULT='mediump'
-     * @property {number} LOW='lowp'
-     * @property {number} MEDIUM='mediump'
-     * @property {number} HIGH='highp'
-     */
-    PRECISION: {
-        DEFAULT: 'mediump',
-        LOW: 'lowp',
-        MEDIUM: 'mediump',
-        HIGH: 'highp'
-    },
+/**
+ * Constants that identify shapes, mainly to prevent `instanceof` calls.
+ *
+ * @static
+ * @constant
+ * @memberof PIXI
+ * @type {object}
+ * @property {number} POLY
+ * @property {number} RECT
+ * @property {number} CIRC
+ * @property {number} ELIP
+ * @property {number} RREC
+ */
+export const SHAPES = {
+    POLY: 0,
+    RECT: 1,
+    CIRC: 2,
+    ELIP: 3,
+    RREC: 4,
+};
 
-    /**
-     * Constants that specify the transform type.
-     *
-     * @static
-     * @constant
-     * @type {object}
-     * @property {number} DEFAULT=STATIC
-     * @property {number} STATIC
-     * @property {number} DYNAMIC
-     */
-    TRANSFORM_MODE:{
-        DEFAULT:    0,
-        STATIC:     0,
-        DYNAMIC:    1
-    },
+/**
+ * Constants that specify float precision in shaders.
+ *
+ * @static
+ * @constant
+ * @memberof PIXI
+ * @type {object}
+ * @property {number} DEFAULT='mediump'
+ * @property {number} LOW='lowp'
+ * @property {number} MEDIUM='mediump'
+ * @property {number} HIGH='highp'
+ */
+export const PRECISION = {
+    DEFAULT: 'mediump',
+    LOW: 'lowp',
+    MEDIUM: 'mediump',
+    HIGH: 'highp',
```

Comment:
Why are the values sometimes white spaced aligned (e.g, `TRANSFORM_MODE` below) and some are not, like this? Seems inconsistent. Also some white-spaced value aligned objects have  have a single space and some have lots of spacing.